ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behaviour
```

Comment:
I think this issue template works for bugs, but not for actual feature requests. Maybe we should restructure the template a bit to have one for bugs and one for stories.

Reply:
Depends on how you look at this. An expected behaviour can also be something new. Any proposal for the story template?

Reply:
What about the following:

### User Story
<!--- Add a user story in the format -->
<!--- As a [user], I want [goal] so that [reason]. -->

### Description & Motivation
<!--- Describe the feature and name the business value of it. -->

### Deliverables
<!--- What needs to be done? Example: -->
<!--- Code changes -->
<!--- Documentation update -->
<!--- Follow up issues -->

### Acceptance Criteria
<!--- A list of criteria to be used as requirements for implementation. Example: -->
<!--- Multiple languages are supported -->
<!--- The description field accepts and verifies HTML -->
<!--- Maximum allowed time for responding is 200ms. -->

### Verification Steps
<!--- Steps used to verify this story, including instruction for setup. -->